### DISCUSSION TITLE
Drag-and-drop sorting of Snippets

### DISCUSSION THREAD

**Discussion by kiyoshi-satoo** [wagtail/wagtail#12096] [Q&A]
Hi, Is there any solution for drag-and-drop sorting of Snippets like Page's "Sort Menu Order".
[wagtail-orderable](https://github.com/elton2048/wagtail-orderable) is not suitable because wagtail-admin is deprecated.

**Comment by gasman** [comment]
Not at the moment, no. There's an open issue for this: https://github.com/wagtail/wagtail/issues/10816

**Comment by SebCorbin** [comment]
Here's what I've done to achieve that:
- make your snippet depend on Orderable
- set up a custom view to your snippet viewset's urlpatterns that will receive snippet instance id in url and position as GET, make it update the sort_order field
- as for the UI, you need to make an IndexView highly inspired from ExplorerIndexView
   - you have to make a custom Table class that will use the OrderableController in JavaScript (bound to the view you created)
   - in the template make your new table be wrapped by a form (no action needed) with a csrf_token

**Reply by kiyoshi-satoo** [reply]
Thanks for the answer. I'll try this one
